tests/app/test_mock_party.py

```diff
@@ -142,7 +140,7 @@ def test_message_by_id_replaces_uuids(self):
                 'ru_id': 'f1a5e99c-8edf-489a-9c72-6cabe6c387fc',
                 'survey': constants.BRES_SURVEY}
 
-        encrypted_token = self._generate_encrypted_token()
+        encrypted_token = _generate_encrypted_token()
```

Comment:
Seems fair enough to me . But this change did not introduce that , so I suggest we put in a ticket to cover that work .